plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreRole.java

```diff
@@ -364,6 +365,23 @@ private void loadExtraConfigParameters(@NotNull DBRProgressMonitor monitor) thro
         }
     }
 
+    private String quoteParamIfNeed(String parameter) {
```

Comment:
Maybe it would be better to extract just the value here, without quotes, and store both the name and the value inside `PostgreRoleSetting`?

Reply:
But anyway, we need the logic about quoting. Your idea to move that into `PostgreRoleSetting`?

Reply:
No, process it here but store key/value separately in `PostgreRoleSetting`